Assets/BossRoom/Scripts/Game/Server/Entity/SpawnNetworkObjectAtLocation.cs

```diff
@@ -0,0 +1,36 @@
+using System.Collections.Generic;
+using Unity.Netcode;
+using UnityEngine;
+using UnityEngine.SceneManagement;
+
+namespace Unity.Multiplayer.Samples.BossRoom.Game
+{
+    public class SpawnNetworkObjectAtLocation : NetworkBehaviour
+    {
+        [SerializeField]
+        GameObject m_NetworkObjectPrefab;
+
+        public override void OnNetworkSpawn()
+        {
+            if (!IsServer)
+            {
+                return;
+            }
+
+            NetworkManager.SceneManager.OnLoadEventCompleted += SpawnNetworkPrefab;
```

Comment:
This gets executed every time a new additive scene is loaded as well, which means that if a character goes near the door, then back to the spawn, then back to the door, etc. there will be multiple objects being spawned at the same place. Was that the intended behaviour?

Reply:
Right. I was doing most of my tests inside just the entrance bit and hadn't thought about the scene being reloaded multiple times. This does overlap with the bug that I've logged [here](https://jira.unity3d.com/browse/MTT-3615). We have a workaround now for parenting in-scene placed NetworkObjects, so if they're part of the scene through NGO, we won't have this issue. I'll add this as a comment on that Jira ticket.